subsys/bluetooth/controller/ll_sw/nordic/lll/lll.c

```diff
@@ -437,6 +440,27 @@ u32_t lll_radio_is_idle(void)
 	return radio_is_idle();
 }
 
+u32_t lll_radio_tx_pwr_get(void) 
+{
+	return radio_tx_power_get();
+}
+
+u32_t radio_txp;
```

Comment:
Global variable not here. Start using bottom up, example, declare in ```struct lll_adv``` ```struct lll_conn``` etc. use this in individual states/role, then move up defining a private interface that satisfies both downstream and upstream (HCI, Application API) modules.